contracts/poseidon_host_bench/.gitignore

```diff
@@ -0,0 +1,9 @@
+# Ignore build artifacts from the local tests sub-crate.
```

Comment:
don't we already have one top-level gitignore to rule them all?

Reply:
It was created by `cargo contract new`. I overlooked that.